### ISSUE TITLE
Freshly made project via rete-kit throws build error

### ISSUE THREAD

**Issue by CZ-DannyK** [retejs/rete#645]
Hey,

i have just tried to make new project via rete-kit (1.4.0) with angular stack (16.1.5) and basically all plugins except non angular renderer ones and when i try to run via ng serve, i get this error:

> Error: src/app/rete/3d.ts:218:14 - error TS4111: Property 'translateY' comes from an index signature, so it must be accessed with ['translateY'].
218   gridHelper.translateY(-320);

I am not sure if is it bug, misconfiguration (like strict) or something else.

I will appreciate help

Thanks,
Daniel

**Comment by Ni55aN** [comment]
`ng serve` or `npm run start`?

**Comment by CZ-DannyK** [comment]
Both

**Comment by Ni55aN** [comment]
just checked on rete-kit v1.4.0 with angular 16, `npm run start` works correctly

**Comment by CZ-DannyK** [comment]
Nope, it doesnt work to me even with `npm run start`, just checked again just to be sure. I can pack my generated project and post link here to check. I havent touch anything.

It seems related to Three.js in 3D template (which i can basically drop and remove referenced to get it working as i dont need that one - should be enough to get it working i hope).

**Comment by Ni55aN** [comment]
what is your node.js version?

**Comment by CZ-DannyK** [comment]
Node - v18.16.0
NPM - 9.5.1

**Comment by Ni55aN** [comment]
OS?

**Comment by CZ-DannyK** [comment]
Windows 11 Pro 22H2
